truffle/src/main/java/org/jruby/truffle/pack/nodes/read/ReadDoubleNode.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I would group these "read primitive" nodes in a single file if that makes sense so to avoid *n* tiny files and ease possible refactoring.